### PR TITLE
task/add-lib-version-to-get-version

### PR DESCRIPTION
**Pull request by laryssesavanna**
Implementing function to validate lib versions

### CODE REVIEW

File: src/lib/orionld/serviceRoutines/orionldGetVersion.cpp [hunk 1]
```diff
@@ -39,7 +39,14 @@ extern "C"
 #include "orionld/common/branchName.h"                         // ORIONLD_BRANCH
 #include "orionld/serviceRoutines/orionldGetVersion.h"         // Own Interface
 
-
+#include "/usr/include/boost/version.hpp"                      // BOOST_LIB_VERSION
+#include "/usr/local/include/microhttpd.h"                     // MHD_VERSION (returns a number)
+#include "/usr/include/openssl/opensslv.h"                     // OPENSSL_VERSION_NUMBER
+#include "/usr/local/include/mongo/version.h"                  // MONGOCLIENT_VERSION
+#include "/usr/include/rapidjson/rapidjson.h"                  // RAPIDJSON_VERSION_STRING
```
Comment (kzangeli) [inline src/lib/orionld/serviceRoutines/orionldGetVersion.cpp]: Include system headers with "<>", and remove /usr/include" from the path

Comment (kzangeli) [reply src/lib/orionld/serviceRoutines/orionldGetVersion.cpp]: E.g. #include <boost/version.hpp>

Comment (kzangeli) [reply src/lib/orionld/serviceRoutines/orionldGetVersion.cpp]: Fixed in 0207b286534bbab5cbc493a919e737d6f9ab29fa

**Comment by kzangeli** [comment]
LGTM

File: src/lib/orionld/serviceRoutines/orionldGetVersion.cpp (hunk 1 above)
Comment (laryssesavanna) [reply src/lib/orionld/serviceRoutines/orionldGetVersion.cpp]: Fixed in 0207b28

**Pull request merged**